fission/src/mirabuf/MirabufSceneObject.ts

```diff
@@ -37,6 +49,22 @@ class MirabufSceneObject extends SceneObject {
         }
 
         this._debugBodies = null
+
+        // creating transform gizmos in TransformGizmos handler
+        this.transformGizmos = new TransformGizmos(
+            new THREE.Mesh(
+                new THREE.SphereGeometry(3.0),
+                new THREE.MeshBasicMaterial({ color: 0x000000, transparent: true, opacity: 0 })
+            )
+        )
+        this.transformGizmos.AddMeshToScene()
+        this.transformGizmos.CreateGizmo("translate")
+
+        // disabling physics initially for all rigid node bodies
+        this._mirabufInstance.parser.rigidNodes.forEach(rn => {
```

Comment:
Please turn adding and removing the gizmos and essentially "putting the mirabuf object in move mode" into a separate function for reusability